pkg/process/monitor/process_monitor.go

```diff
@@ -239,8 +239,10 @@ func (pm *ProcessMonitor) Initialize() error {
 	pm.initOnce.Do(
 		func() {
 			pm.done = make(chan struct{})
+			pm.callbackRunnersWG = sync.WaitGroup{}
```

Comment:
Can you remove this line (and the one below)? I don't think it's idiomatic to initialize types to their zero values

Reply:
this is "reinitialization", as in the tests (unlike the normal scenario) we can re-initialize the process monitor
does that make sense?

Reply:
See the full (current or new) implementation of the `Stop` function, it contains explanation for the "reinitialization" phase
https://github.com/DataDog/datadog-agent/blob/2eb141c6adc802bc60152a7b29af442508a27e78/pkg/process/monitor/process_monitor.go#L316-L342